pkg/sns/composer.json

```diff
@@ -9,7 +9,7 @@
         "php": "^7.4|^8.0",
         "queue-interop/queue-interop": "^0.8",
         "enqueue/dsn": "^0.10",
-        "aws/aws-sdk-php": "~3.155"
+        "aws/aws-sdk-php": "~3.319"
```

Comment:
```suggestion
        "aws/aws-sdk-php": "~3.155"
```

Reply:
I change to use he same version in the global composer